src/components/ConnectWallet/index.tsx

```diff
@@ -13,7 +13,11 @@ interface WalletProps {
 export default function Wallet({ disabled, onClickConnectWallet }: WalletProps) {
   // Attempt to connect eagerly on mount
   useEffect(() => {
-    connections.forEach(([wallet, _]) => (wallet.connectEagerly ? wallet.connectEagerly() : wallet.activate()))
+    connections.forEach(([wallet, _]) =>
+      wallet.connectEagerly
+        ? wallet.connectEagerly(defaultChainId)?.catch((e) => console.log(e))
```

Comment:
how do we surface errors to devs in general?

Reply:
devs can pass in a "onError" prop that gets handled by our ErrorBoundary, but choosing to just log these to console since it's fine if we can't connect a wallet eagerly

more on handling errors in this PR:
https://github.com/Uniswap/widgets/pull/55/files